File: py i cpp/warunek.py
def main(args):
    a = int(input('Podaj pierwszą liczbę: '))
    b = int(input('Podaj drugą liczbę: '))
    c = int(input('Podaj trzecią liczbę: '))
 
    if a > b and a > c:
        print('Największa liczba: ', a)
    elif b > a and b > c:
        print('Największa liczba: ', b)
    else:
        print('Największa liczba: ', c)
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: py i cpp/warunek.cpp
#include <iostream>
using namespace std;

int main(int argc, char **argv)
{
	float a, b, c;
    a = b = c = 0;
    
    cout << "Podaj pierwszą liczbę: ";
    cin >> a;
    cout << a << endl;
    
    cout << "Podaj drugą liczbę: ";
    cin >> b;
    cout << b << endl;
    
    cout << "Podaj trzecią liczbę: ";
    cin >> c;
    cout << c << endl;
    
    if ((a > b) && (a > c)) {
        cout << "Największa liczba to: " << a;
    }
    else if((b > a) && (b > c)) {
        cout << "Największa liczba to: " << b;
    }
    else {
        cout << "Największa liczba to: " << c;
    }
    
	return 0;
}

